Sources/Sampler/py_sampler.hpp

```diff
@@ -152,6 +161,94 @@ void AddSamplerModule(py::module& m) {
   AddExactSampler(subm);
   AddCustomSampler(subm);
   AddMetropolisHastings(subm);
+
+
+  py::class_<MCResult>(subm, "MCResult",
```

Comment:
this ```MCResult``` type also seems redundant, see my previous reasoning

Reply:
Yeah, I think I kept it by accident. it is not actually used in Python anymore.

Reply:
Can we remove it then? :)

Reply:
I wouldn't object to that. But I also do not see the need to do this here. (As long as the Python interface is reasonably stable, this can be changed under the hood later.)